### PR TITLE
Translate Lang es_es.lang

### PR DESCRIPTION
**Pull request by Yataro-Ibuza**
This is a Lang File with a translate to Spanish (Spain)

### CODE REVIEW

File: src/main/resources/assets/chisel/lang/es_es.lang [hunk 1]
```diff
@@ -0,0 +1,2871 @@
+item.chisel.chisel_iron.name=Cincel
+item.chisel.chisel_diamond.name=Cincel de Diamante
+item.chisel.chisel_hitech.name=iChisel
+item.chisel.offsettool.name=Varita de Diseños Ender
```
Comment (amadornes) [inline src/main/resources/assets/chisel/lang/es_es.lang]: What about "Varita de Desplazamiento de Ender"?
I'd say it fits better with the original "Ender Offset Wand"
(Also propagate this change to L56)

File: src/main/resources/assets/chisel/lang/es_es.lang [hunk 2]
```diff
@@ -0,0 +1,2871 @@
...
+
+chisel.spawnwither=Un Cincel puede ser algo peligroso...
+
+item.chisel.chisel.desc.gui=%sClic derecho%s para abrir la Interfaz
+item.chisel.chisel.desc.lc1=%sClic izquierdo%s para cincelar un bloque del mundo
+item.chisel.chisel.desc.lc2=%sSelecciona el objeto%s al tenerlo en el inventario
```
Comment (amadornes) [inline src/main/resources/assets/chisel/lang/es_es.lang:10]: I'd replace "objeto" with "bloque" (originally "block") and "al tenerlo" with "colocándolo" (originally "by leaving it")
(Also propagate this change to L51, L54 & L60)

**Review by amadornes — COMMENTED** [review]
It seems like an overall solid translation, though I'd make a few changes here and there, so I've left a couple of comments with suggested alt translations.
Although the originals aren't really wrong, the ones I suggested sound better to me in the context of the mod. Feel free to change them, or not.

There's the review you asked for, @tterrag1098 ;P

File: src/main/resources/assets/chisel/lang/es_es.lang (hunk 2 above)
Comment (Yataro-Ibuza) [reply src/main/resources/assets/chisel/lang/es_es.lang:10]: Ohhh, thanks for those detail, i´ll change the Ender Offset Wand by "Varita de Desplazamiento de Ender",

and the second detail i´ll leave it as you modified.

Thanks a lot for inspect it!

File: src/main/resources/assets/chisel/lang/es_es.lang (hunk 1 above)
Comment (Yataro-Ibuza) [reply src/main/resources/assets/chisel/lang/es_es.lang]: Okay! then i´ll change it as soon as possible

**Comment by tterrag1098** [comment]
Thanks for the assist @amadornes :D

**Comment by tterrag1098** [comment]
@Yataro-Ibuza as soon as you implement all of his suggestions, I will merge this.

File: src/main/resources/assets/chisel/lang/es_es.lang (hunk 2 above)
Comment (Yataro-Ibuza) [reply src/main/resources/assets/chisel/lang/es_es.lang:10]: Also, i make the changes that you mentioned it, so now the file has all the changes

**Comment by Yataro-Ibuza** [comment]
@tterrag1098 i have already implement all the suggestions, so now you can merge this PR.

And thanks either @amadornes!

**Pull request merged**